endtoendtests/Azure.Functions.Java.Tests.E2E/Azure.Functions.Java.Tests.E2E/Azure.Functions.Java.Tests.E2E.csproj

```diff
@@ -11,6 +11,7 @@
     <PackageReference Include="Microsoft.Azure.EventHubs" Version="2.2.1" />
     <PackageReference Include="Microsoft.Azure.Management.EventGrid" Version="6.0.0" />
     <PackageReference Include="Microsoft.Azure.EventGrid" Version="3.2.0" />
+    <PackageReference Include="Microsoft.Azure.ServiceBus" Version="5.0.0" />
```

Comment:
Nit, why you use this version? Since the service bus extension seems 4.2.0 https://github.com/Azure/azure-functions-servicebus-extension/blob/dev/src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/WebJobs.Extensions.ServiceBus.csproj#L41 

If you don't care, it works, so no problem.

Reply:
This is the ServiceBus version not the extension version. This is the reason to have 5.0.0